src/tests/ftest/erasurecode/offline_rebuild_aggregation.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+import time
+from ec_utils import ErasureCodeIor, check_aggregation_status
+from apricot import skipForTicket
+
+class EcodAggregationOffRebuild(ErasureCodeIor):
+    # pylint: disable=too-many-ancestors
+    """
+    Test Class Description: To validate Erasure code object for offline rebuild
+                        with different Aggregation mode for Partial strip data.
+    :avocado: recursive
+    """
+    def test_ec_offline_rebuild_agg_disabled(self):
+        """Jira ID: DAOS-7313.
+
+        Test Description: Test Erasure code object aggregation disabled mode
+                          with IOR.
+        Use Case: Create the pool, disabled aggregation, run IOR with supported
+                  EC object type with partial strip.
+                  Verify that Aggregation should not triggered.
+                  Verify the IOR read data at the end.
+                  Kill single server and wait for rebuild.
+                  Read and verify all the data.
+                  Kill second server and wait for rebuild.
+                  Read and verify data with +2 Parity with no data corruption.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,large,ib2
+        :avocado: tags=ec,aggregation,ec_array,ec_aggregation
+        :avocado: tags=ec_offline_rebuild_agg_disabled
+        """
+        # Disable the aggregation
+        self.pool.set_property("reclaim", "disabled")
+        self.pool.connect()
+        self.log.info("pool_percentage Before = {} ".format(
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions